anagrafica/viste.py

```diff
@@ -1660,9 +1660,16 @@ def profilo(request, me, pk, sezione=None):
 @pagina_privata
 def presidente(request, me):
     sedi = me.oggetti_permesso(GESTIONE_SEDE)
-    contesto = {
-        "sedi": sedi
-    }
+    sedi_presidente = sorted(me.oggetti_deleghe(), key=lambda x: x.get_level())
```

Comment:
@yakky hm, puoi evitare l'ordinamento dinamico e quello slice `[0]` con.

```python3
sede_presidente = sedi.first('level')  # None o Sede.
```

Se puoi inoltre inizializza sempre il contesto a dizionario vuoto a inizio vista. Se ci operi dentro un if, usa sempre `.update`, cosi' da non dover andare a modificare il tuo codice se si vuole aggiungere del contenuto statico in contesto, o simile.